turte_graphics/turtle.js

```diff
@@ -0,0 +1,111 @@
+class Turtle {
+  constructor(x, y) {
+    this.x = x;
+    this.y = y;
+    this.position = [x, y];
+    this.path = [[x, y]];
+    this.direction = "x";
+  }
+
+  forward(steps) {
+    for (let i = 0; i < steps; i++) {
+      if (this.direction === "x") {
+        this.x++;
+      } else if (this.direction === "-y") {
+          this.y--;
+      } else if (this.direction === "-x") {
+        this.x--;
+      } else {
+          this.y++;
+      }
+      this.position = [this.x, this.y];
+      this.path.push(this.position);
+    }
+    return this;
+  }
+
+  right() {
+    if (this.direction === "x") {
+      this.direction = "-y";
+    } else if (this.direction === "-y") {
+      this.direction = "-x";
+    } else if (this.direction === "-x") {
+      this.direction = "y";
+    } else {
+      this.direction = "x";
+    }
+    return this;
+  }
+
+  left() {
+    if (this.direction === "x") {
+      this.direction = "y";
+    } else if (this.direction === "-y") {
+      this.direction = "x";
+    } else if (this.direction === "-x") {
+      this.direction = "-y";
+    } else {
+      this.direction = "-x";
+    }
+    return this;
+  }
+
+  print() {
+    const full = '\u2588';
```

Comment:
Try to be consistent with your use of `'` or `"`

Reply:
done